src/components/SpacemeshApiClientMock/index.tsx

```diff
@@ -41,16 +41,17 @@ const SpacemeshApiClientMock = <P extends IProps>(Component: React.ComponentType
         /**
          * Broadcast a transaction to the Spacemesh network
          * @param tx raw transaction to broadcast
+         * @param hash the transaction hash. 
```

Comment:
hmm... this changes the API so the mock now has a different interface.  I prefer we don't break API compatibility and  add a class property: `nextResult`, give it an initial value ('0xe670ec...') and return its value wherever the mock return a result.  This way we can keep this function's interface free of Mock-specific parameters and give the use the abilit to set the property to a different value if the test requires it.